packages/notifier/src/types-ambient.js

```diff
@@ -50,19 +62,48 @@
 
 /**
  * @template T
- * @typedef {ERef<PublicationRecord<T>>} PublicationList
- * Will be shared between machines, so it must be safe to expose. But other
- * software should consider it opaque, not depending on its internal structure.
+ * @typedef {Promise<PublicationRecord<T>>} PublicationList
+ * Will be shared between machines, so it must be safe to expose. But software
+ * outside the current package should consider it opaque, not depending on its
+ * internal structure.
```

Comment:
Is PublicationList carrying its own weight? We might be better off just dropping it in favor of only PublicationRecord.